pages/landing.tsx

```diff
@@ -0,0 +1,277 @@
+import { useTranslation } from 'next-i18next'; // Translation hook
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import {
+  Text,
+  Divider,
+  GridItem,
+  Box,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Link,
+  UnorderedList,
+  ListItem,
+  Button,
+} from '@chakra-ui/react'; // Chakra UI
+import Layout from '@components/applicant/Layout'; // Layout wrapper
+
+export default function Landing() {
+  const { t } = useTranslation('common');
+
+  return (
+    <Layout>
+      <GridItem colSpan={8} colStart={1} mt="64px">
+        <Text as="h1" textStyle="display-xlarge" align="left">
+          {t('landing')}
+        </Text>
+        <Text as="p" textStyle="display-small" align="left" mt="24px">
+          Find the services you need related to your British Columbia accessible parking permit. You
+          may chose to use our online service or download a physical form and email, mail or drop it
+          off in person to RCD.{' '}
+        </Text>
+      </GridItem>
+      <GridItem colSpan={5} colStart={1} mt="28px">
+        <Text as="h2" textStyle="display-medium" align="left" fontWeight="semibold">
+          Option 1: Online Service
+        </Text>
+        <Text as="p" textStyle="body-regular" align="left" mt="24px">
+          You will need to complete the following steps for our online services
+        </Text>
+        <UnorderedList mt="12px" pl="10px">
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Provide your user ID (available on your wallet card)</Text>
+          </ListItem>
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Provide the last 4 digits of your phone number</Text>
+          </ListItem>
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Provide your date of birth</Text>
+          </ListItem>
+          <ListItem textAlign="left" textStyle="body-regular">
+            <Text as="p">Complete the online form and pay a $26 processing fee</Text>
+          </ListItem>
+        </UnorderedList>
+      </GridItem>
+      <GridItem colSpan={5} colStart={7} mt="28px">
+        <Text as="h2" textStyle="display-medium" align="left" fontWeight="semibold">
+          Option 2: Download Form
+        </Text>
+        <Text as="p" textStyle="body-regular" align="left" mt="24px">
+          If you would prefer to fill out a replacement form for a lost or stolen parking permit by
+          hand, or want to request a new parking permit, please download the appropriate form below.
+          <br></br>
+          <br></br>
```

Comment:
In favour of semantic HTML (to meet accessibility requirements), we want to avoid using `<br />` to split paragraphs. Instead, this should be 2 `Text` components with a margin between them.